### DISCUSSION TITLE
Malicious activity - production deployment down - perpetrator IP addresses captured - Cannot submit a case!?

### DISCUSSION THREAD

**Discussion by nathanjhood** [vercel/community#4298] [Account Issue]
### Question

Hi all

I would please like to ask if your security-involved department can review the domain logs from my 'stoneydsp.com' domain that I have configured via your nameservers on my org account - my org being "StoneyDSP".

I have been silently dealing with malicious scraper attacks on my GitHub repos for a really long time, more than one year, and then once I launched a public static HTML page on your platform, they descended upon it like sharks to blood. I had to keep dumbing myself down until they got sloppy, laid some booby traps, and now I have everything I need on record (they bugged my laptop and possibly my phone also). 

Doing so, I have had to take down my production deployment permanently until this issue is resolved.

Can I please also ask for some advice about guarantees that my deployments are and will be secure on your platform? This group of bad actors - who are employees and alumni of \<censored\>, no less - has been incredibly persistent at attempting to POST and manipulate every tiny little security vulnerability, and unfortunately I'm very little experienced at all in Web Dev...

I tried raising a ticket on your platform AND have made contact several times with your security team by email, but nothing seems to be getting through. If for some reason you cannot see all this activity on all of my account logs, no worries - I have multiple copies of everything.

Thanks and sorry for the contact on the weekend.

Best, Nathan/StoneyDSP

**Comment by amyegan** [comment]
Hi @nathanjhood. I'm sorry to hear about the malicious attacks on your repos. My first suggestion would have been to [contact the Security team](https://vercel.com/security) or open a ticket using the [support form](https://vercel.com/help#issues) or other [direct channels](https://vercel.com/guides/how-to-get-vercel-support#direct-channels), but it sounds like you already did that. 

Tickets are processed by the order in which they are received. At this point, it's a matter of waiting for the security support team to reach your ticket. You can find more info about response times at https://vercel.com/guides/vercel-support-queue-time
